interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py

```diff
@@ -167,11 +167,19 @@ def __init__(self, dim):
 
 	
 	def set(self, field, value, idx=None):
+		# cast to np array
 		if type(value) is not np.ndarray:
-			if type(value) is int:
+			if (type(value) is int) or (type(value) is float):
```

Comment:
why setting a dim as float?

Reply:
As far as I understand, this is not only a setter for dims but also for bound values, thus floats make sense.